### PR TITLE
Added in a handful of quality React.js learning materials

### PR DESCRIPTION
**Pull request by wesbos**
I've spent the last half year building with and teaching React - I've evaluated a number of materials, created my own and I've got my ear to the ground hearing from students what other materials they have found useful.

This isn't a comprehensive list, but they are the ones that come up over and over again.

### CODE REVIEW

**Comment by petehunt** [comment]
Thanks! I'll take a look at these later this week. Ideally I'd like to find a way to work these into the existing sections, since the emphasis of this document is to go step-by-step, so when I take a look I'll be doing it with that in mind!

Again -- thanks so much for these resources.

**Comment by wesbos** [comment]
No problem - let me know your email and I can get you access to react for beginners if you want to vet the content

**Comment by petehunt** [comment]
pete at smyte dot com. thanks!

**Comment by srameshr** [comment]
Still cant see this on the master ```README.md```. 
Lots of dev's are going to stumble upon this and they would be up fast and running with the <b>React for beginners</b> videos.

**Comment by bebraw** [comment]
@petehunt Do you want to consider including [survivejs](https://github.com/survivejs/webpack_react)? I know it's not on the same level as Wes' excellent material, but it might be worth mentioning.

**Comment by wesbos** [comment]
@petehunt I sent you an account.

File: README.md [hunk 1]
```diff
@@ -84,3 +84,13 @@ If you have decided to use Flux, the most popular and well-documented Flux libra
 ## Learning Relay, Falcor etc
 
 These are technologies that help you reduce the number of AJAX requests. They’re still very cutting-edge, so if you don’t have a problem with too many AJAX requests, you don’t need Relay or Falcor.
+
+## Courses and Learning Material
+
+* [React For Beginners](https://ReactForBeginners.com) [watch]
+* [EggHead.io React Tutorials](https://egghead.io/technologies/react) [watch]
```
Comment (kentcdodds) [inline README.md]: I think this should be "Egghead.io" not "EggHead.io"

**Comment by wesbos** [comment]
@petehunt Any chance you checked out these resources?

**Comment by mjackson** [comment]
TBH this PR feels like it's adding marketing to a document that previously was just trying to be informative. I'd personally probably be less enthusiastic about recommending this document if it started adding these sorts of things. It's just advertising.

**Comment by wesbos** [comment]
So if we take the paid ones out, then it's okay? If it's good way for people to learn, I don't see why it can't be recommended. 

I'm okay either way - my intention of this wasn't to make it marketing. I get asked all the time for the best resources on learning React and I'm happy to point to paid or free.

@petehunt any thoughts?

**Comment by atuttle** [comment]
I don't have a problem with paid resources as long as they're clearly marked (e.g. 💰) and don't constitute the majority of what's listed.

**Comment by wesbos** [comment]
FrontEnd Handbook does a good job at this - http://www.frontendhandbook.com/learning/javascript.html

**Pull request closed without merging**